src/WebJobs.Script.Grpc/WebJobs.Script.Grpc.csproj

```diff
@@ -19,14 +20,13 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="Google.Protobuf" Version="3.11.4" />
+    <PackageReference Include="Grpc.AspNetCore" Version="2.32.0" />
     <PackageReference Include="Google.Protobuf.Tools" Version="3.11.4" />
-    <PackageReference Include="Grpc.Core" Version="2.27.0" />
-    <PackageReference Include="Grpc.Tools" Version="2.27.0">
+    <PackageReference Include="Grpc.Tools" Version="2.32.0">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.9" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="3.1.9" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.0-beta004">
       <PrivateAssets>all</PrivateAssets>
```

Comment:
I will update our protobuf generation logic to use https://docs.microsoft.com/en-us/aspnet/core/grpc/dotnet-grpc?view=aspnetcore-3.1#add-references available in the managed grpc library, instead of trunning `generate_protos` scripts.

Reply:
Will you be sending a separate PR for removing generate_protos script?

Reply:
I updated this PR with the change. The only reason I hadn't because I haven't figured out a proper way to specify the output directory for specific files after conversion. For example, the way we have `Messages/DotNet/FunctionRpc.cs` and `Messages/DotNet/FunctionRpcGrpc.cs` I couldn't get that with the new target way.

The Target has an option for `OutputDir` but that causes issues with other ".rpc" helper files. It doesn't have something like `OutputFiles` similar to `Outputs` that we use today.

With the current change, the converted files don't show up in `Messages/DotNet`. Is there a reason we need that? Or is it just for convenience? If we do, then I can revert my target change and take that separately to not block this work.

Reply:
Folder structure is OK. As long as namespace remains the same this change is OK. Thanks.